### ISSUE TITLE
Multi-page PDF from single page HTML

### ISSUE THREAD

**Issue by Paktas** [eKoopmans/html2pdf#43]
We generate 1 page PDF and users need to print 2 copies of it. They need to adjust the printing quantity from the typical "1" to "2".
But very often users forget to do that and print 1 copy instead of 2. Therefore would love to show a 2 page PDF instead of 1.
Content would be exactly the same on both pages, but I don't want to duplicate HTML so that in case of editing we wouldn't need to edit it in 2 locations on the same HTML.

**Comment by eKoopmans** [comment]
Hey @Paktas, really sorry I never got back to you! It's an interesting request. As a long-term goal I may try to create a "Print Preview" option, in which case quantity could be a setting, but to be honest even Chrome's print dialog doesn't give you a quantity option when saving to PDF.

The simplest approach would probably be to get the jsPDF object at the end of html2pdf, add a page, then reuse that object in a second call to html2pdf. I haven't created a version of html2pdf that would allow that yet (reusing an existing jsPDF object), but it's in the works. There have been a few requests for an updated API and that would be one of the added features.

So all that to say - it's not possible yet, and I probably won't add a specific "quantity" option any time soon, but with the updated API it should be fairly easy to customize yourself to get it working. I'll add this to the "updated API" project tracking, which is my next goal for this package.

**Comment by Paktas** [comment]
Thanks. Makes sense. Will follow Updated API ticket to work a workaround for this.

**Comment by eKoopmans** [comment]
Hi, I have just pushed a new branch, [new-api](https://github.com/eKoopmans/html2pdf/tree/new-api), which is ready for testing! [The readme](https://github.com/eKoopmans/html2pdf/tree/new-api#advanced-usage) has all the information on the new API. Please let me know if there are any problems with it - I want to make sure it's stable before merging into master!

Regarding this specific request, you should now be able to achieve it like so:
```js
html2pdf().from(targetElement).toPdf().get('pdf').then(function (pdf) {
  pdf.addPage();
}).toPdf().save();
```

**Comment by eKoopmans** [comment]
Hi, closing as resolved by the v0.9.0 new API.
